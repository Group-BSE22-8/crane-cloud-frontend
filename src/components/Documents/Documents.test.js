import React from 'react';
import { shallow } from "enzyme";
import Privacy from './privacy';
import Terms from './terms';


describe('Test the privacy document', () => {
    it('checks if the privacy document matches the snapshot', () => {
        const PrivacyDoc = shallow(<Privacy />);
        expect(PrivacyDoc).toMatchSnapshot();
    });
});

describe('Test the terms document', () => {
    it('checks if the terms document matches the snapshot', () => {
        const TermsDoc = shallow(<Terms />);
        expect(TermsDoc).toMatchSnapshot();
    });
});