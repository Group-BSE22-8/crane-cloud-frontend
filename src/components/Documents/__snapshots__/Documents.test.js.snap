// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the privacy document checks if the privacy document matches the snapshot 1`] = `ShallowWrapper {}`;

exports[`Test the terms document checks if the terms document matches the snapshot 1`] = `ShallowWrapper {}`;
